postorius/mailman-web/settings.py

```diff
@@ -104,8 +101,17 @@
     'postorius.middleware.PostoriusMiddleware',
 )
 
+# Use old-style Middleware class in Python 2 and released versions of
+# Django-mailman3 don't support new style middlewares.
+
+if sys.version_info < (3, 0):
```

Comment:
I think we can safely remove this and just use the MIDDLEWARE settings since we don’t support Python 2 anymore. It was needed in old times when we supported Python 2.7

This changeset is really not much use, we should just use `MIDDLEWARE` settings instead.

Reply:
Done @maxking